packages/@aws-cdk/aws-ec2/lib/security-group.ts

```diff
@@ -229,6 +229,8 @@ export interface SecurityGroupProps {
    * @default true
    */
   readonly allowAllOutbound?: boolean;
+
+  readonly disableInlineRules?: boolean;
```

Comment:
Currently this is being handled by options.  This is not the final design, but this allowed me to write some unit tests.

Reply:
Yap. This is actually not bad to keep; I think adding a context variable to configure this globally across the entire project ALSO makes sense, but configuring for individual SGs is definitely a good thing to have. Leave it in!

Reply:
Sounds like a plan to me